### PR TITLE
feat: Add new images to src/assets folder

### PR DESCRIPTION
**Pull request by NkwaTambe**

### CODE REVIEW

File: package-lock.json
Comment (stephane-segning) [inline package-lock.json:1]: Please don't use npm but yarn.

Comment (NkwaTambe) [reply package-lock.json:1]: ok sir

Comment (NkwaTambe) [reply package-lock.json:1]: please approve changes sir

File: yarn.lock
[the remark this answers is not in the document]
Comment (NkwaTambe) [reply yarn.lock:1]: yes sir i did that..

yarn build

[the remark this answers is not in the document]
Comment (NkwaTambe) [reply yarn.lock:1]: please sir review new changes

**Review by stephane-segning — APPROVED** [review]

**Pull request merged**